users_ldap_populate/model/users_ldap.py

```diff
@@ -100,12 +101,29 @@ def action_populate(self, cr, uid, ids, context=None):
                     conf['ldap_filter'])
             results = self.get_ldap_entry_dicts(conf)
             for result in results:
+                login = result[1][login_attr][0].lower().strip()
```

Comment:
afaik odoo login name is case sensitive wouldn't it be weird to have to log into odoo with lowercase and the rest of the apps with uppercase or am i missing something here? /cc @gurneyalex

Reply:
in LDAP, the login field is typically case insensitive, and people are expecting this -> if we need to mimick this in odoo. 

The code here is stolen from auth_ldap, and we need to keep this consistent.

Reply:
@leemannd @gurneyalex thanks for the explanation. Makes sense that way.